.vib/argo-cd/goss/argo-cd.yaml

```
@@ -0,0 +1,7 @@
+command:
+  check-app-version:
+    # argocd version exits with exit code 1
+    exec: {{ .Vars.version.bin_name }} {{ .Vars.version.flag }}
+    exit-status: 1
+    stdout:
+    - /.*{{ .Env.APP_VERSION }}.*/
```

AFAIK, this is equivalent to:

```suggestion
    - {{ .Env.APP_VERSION }}
```